speech/system-test/adaptationCustomClassReference.v2.test.js

```diff
@@ -0,0 +1,91 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const {expect} = require('chai');
+const {describe, it} = require('mocha');
+const sinon = require('sinon');
+const uuid = require('uuid');
+const speech = require('@google-cloud/speech').v2;
+
+const {createCustomClassV2} = require('../adaptationCustomClassReference.v2');
+
+const text = 'how old is the Brooklyn Bridge';
+
+let recognizerName, projectId, customClassName, phraseSetName;
```

Comment:
Nit: can these be defined as `const`s here?

Reply:
No, they're not deterministic as the names will depend on the projectId.